packages/messaging/native.js

```diff
@@ -0,0 +1,210 @@
+/**
```

Comment:
Is JSDoc incapable of consuming markdown or similar? This isn't really a JS file.

Reply:
@jonathanKingston you can use markdown, but then you can't do internal links, which are pretty useful here.